src/System.Memory/tests/ReadOnlySpan/IndexOf.char.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    // Adapted from Span/IndexOf.cs
+    public static partial class ReadOnlySpanTests // .IndexOf<Char>
```

Comment:
This has nothing to do with `Contains` but I noticed that `ReadOnlySpan` did not have specific tests for `<char>`. 
So I copied and adapted the ones from `Span<char>`.

Reply:
Right, we had them here before, but they were moved/merged with the string tests to consolidate them in one place and improve coverage while doing so: https://github.com/dotnet/corefx/pull/30176/

Therefore, I don't think this is needed anymore.

Reply:
Deleted